frontend/src/metabase/components/Popover.css

```diff
@@ -20,7 +20,7 @@
   box-shadow: 0 4px 10px var(--color-shadow);
   background-color: var(--color-bg-white);
   border-radius: 6px;
-  /* overflow: hidden; */
+  overflow: hidden;
```

Comment:
@tlrobinson it looks like you commented this out in #10497. Without that the yellow at the top of the popover was creating sharp corners. Any recollection what might break by removing this fence?

Reply:
I don't recall, but if you uncomment it and none of the issues in #10497 regress then go for it.